bucket/openark.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "1.1.0",
+    "description": "An anti-rookit(ARK) tool",
+    "homepage": "https://openark.blackint3.com",
+    "license": "LGPL-2.1-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk64.exe#/openark.exe",
+            "hash": "6fad8b0553d0ab5464f2547b8890696b0bcf4e1d8ea89b29e6c9840389b3a95e"
+        },
+        "32bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk32.exe#/openark.exe",
+            "hash": "550056c99590a902d9df4a2dfbf75da9f045dd8fb25099b30b34f0ca3937a2e1"
+        }
+    },
+    "shortcuts": [
+        [
+            "OpenArk.exe",
+            "OpenArk"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/BlackINT3/OpenArk"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/BlackINT3/OpenArk/releases/download/v$version/OpenArk64.exe#/openark.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/BlackINT3/OpenArk/releases/download/v$version/OpenArk32.exe#/openark.exe"
```

Comment:
It is not `openark.exe` 

![image](https://user-images.githubusercontent.com/13260377/111729864-55ac4500-8870-11eb-968a-0c74d84fe5ac.png)

Reply:
Actually, the windows will ignore the case. 😂 But I will change to it's proper case.